specs/latest/1.0/index.html

```diff
@@ -3362,6 +3362,18 @@ <h2 class="no-toc">Table of contents</h2>
 
 </p>
 
+<div class="note">
```

Comment:
Olli just added the ability to specify the classes "note rationale" to get slightly different formatting. Could you use that here?